estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<odoo>
+  <record id="estate_property_offer_action" model="ir.actions.act_window">
```

Comment:
```suggestion
  <record id="action_estate_property_offer" model="ir.actions.act_window">
```
Always name an action with `action_` followed by its working.
Make sure to check all the actions you have created so far and do the corrections